waddrmgr/scoped_manager.go

```diff
@@ -915,7 +915,7 @@ func (s *ScopedKeyManager) extendAddresses(ns walletdb.ReadWriteBucket,
 	}
 
 	// Derive the appropriate branch key and ensure it is zeroed when done.
-	branchKey, err := acctKey.Child(branchNum)
+	branchKey, err := acctKey.Derive(branchNum)
```

Comment:
An alternative to this would be for us to do a "migration" in order to detect any instances (by scanning the key range), and then picking if we should use the new derivation or not from there. However, this doesn't protect us from the scenario where a user imports from scratch, which puts us back at a new `aezeed` version.

Reply:
Ah, thanks for the explanation. Should be good now.